commons_boundaries/lib.py

```diff
@@ -22,3 +22,12 @@ def proj4_from_shp(shp_path):
         return crs
     else:
         return ''
+
+
+def encoding_from_shp(shp_path):
+    shp_path = Path(shp_path)
+    cpg_path = shp_path.with_suffix('.cpg')
+    if cpg_path.is_file():
+        return open(str(cpg_path)).readline()
+    else:
```

Comment:
It's a very minor style thing, but I think I'd drop the explicit else in cases like this, so the shape is:

```
if foo:
    return 'bar'
return ''
```